onnxruntime/core/graph/model.cc

```diff
@@ -284,6 +284,38 @@ Status Model::Load(std::unique_ptr<ModelProto> p_model_proto, std::shared_ptr<Mo
   return Status::OK();
 }
 
+template <typename T>
+static Status LoadModel(const T& file_path, ONNX_NAMESPACE::ModelProto& model_proto) {
```

Comment:
>static Status [](start = 0, length = 13)

Too many return paths with different messages. Could you, please, check that on Windows you get the same error code as everywhere else?